Symfony/CS/Tests/Fixer/Contrib/PhpUnitConstructFixerTest.php

```diff
@@ -106,5 +78,27 @@ public function provideTestFixCases()
     );',
             ),
         );
+
+        return array_merge(
+            $cases,
+            $this->generateCases('<?php $this->assert%s(%s, $a, "%s", "%s");', '<?php $this->assert%s%s($a, "%s", "%s");'),
+            $this->generateCases('<?php $this->assert%s(%s, $a); //%s %s', '<?php $this->assert%s%s($a); //%s %s')
+        );
+    }
+
+    private function generateCases($fromTemplate, $toTemplate)
```

Comment:
That is why I don't like squashing when it is not needed.
sth goes wrong:
travis-ci.org/FriendsOfPHP/PHP-CS-Fixer/jobs/85576570#L200